Tests/LibTimeZone/TestTimeZone.cpp

```diff
@@ -169,7 +169,7 @@ TEST_CASE(get_time_zone_offset_with_dst)
     test_offset("America/Phoenix"sv, 1671453238, offset(-1, 7, 00, 00), No); // Monday, December 19, 2022 12:33:58 PM
 
     // Moscow's observed DST changed several times in 1919.
-    test_offset("Europe/Moscow"sv, -1609459200, offset(+1, 2, 31, 19), No);  // Wednesday, January 1, 1919 12:00:00 AM
+    test_offset("Europe/Moscow"sv, -1609459200, offset(+1, 3, 31, 19), Yes); // Wednesday, January 1, 1919 12:00:00 AM
```

Comment:
One of the things this set of tests was trying to capture was the transitions from STD -> DST -> STD. If the time for STD was a bit off here, can we pick a time which is in STD?